### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by npmaile**
opening pull request to make sure it works. It probably doesn't at the moment

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -33,20 +33,15 @@ jobs:
...
-
       - name: Install go stringer
-        run: go get golang.org/x/tools/cmd/stringer
+        run: go get golang.org/x/tools/cmd/stringer@latest
 
       # Runs a single command using the runners shell
       - name: Build Linux
         run: make build
 
       # Runs a set of commands using the runners shell
```
Comment (maevyn11) [inline .github/workflows/main.yml:43]: dead comment?

File: Makefile [hunk 2]
```diff
@@ -10,8 +10,8 @@ build: generate frontend
 client-build: generate #build just the golang code without the frontend
 	go build main/phonon.go
 
-windows-build: generate
-	GOOS=windows CGO_ENABLED=1 CC=$(Win-CC) go build main/phonon.go
+windows-build: generate frontend
+	GOOS=windows CGO_ENABLED=1 go build -ldflags "-H=windowsgui" main/phonon.go
```
Comment (maevyn11) [inline Makefile:14]: does this "-H=windowsgui" flag actually manage to pull in the windows systray weird C dependency stuff?

File: Makefile [hunk 3]
```diff
@@ -27,8 +27,8 @@ android-sdk:
 	cd session && gomobile bind  -target android -o ../androidSDK/phononAndroid.aar
 
 frontend:
-	npm --prefix gui/frontend install
-	npm --prefix gui/frontend run build
+	(cd gui/frontend && npm install)
+	(cd gui/frontend && npm run build)
```
Comment (maevyn11) [inline Makefile:31]: any particular reason for this change in building the frontend? Just curious if there's some advantage to cd'ing

**Review by maevyn11 — APPROVED** [review]
Looks good to merge to me, just left a couple comments with general questions and pointed out a dead comment.

File: Makefile (hunk 3 above)
Comment (npmaile) [reply Makefile:31]: windows crap

**Pull request merged**